selenium-server-standalone: restore htmlunit-driver support

hsenag opened the pull request:
It was moved to a separate project between versions 2.45 and 2.53:
https://github.com/SeleniumHQ/selenium/commit/2d3150b

###### Motivation for this change

This restores the previous behaviour of selenium-server-standalone, namely having the HtmlUnit driver available. They've been split into separate projects upstream, but it still makes sense for end users to have the whole package. selenium-server-standalone already depended on chromedriver in a similar way.

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [X] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Mic92 commented on pkgs/development/tools/selenium/htmlunit-driver/default.nix:
```diff
@@ -0,0 +1,29 @@
+    homepage = https://github.com/SeleniumHQ/htmlunit-driver;
+    description = "A WebDriver server for running Selenium tests on the HtmlUnit headless browser";
+    platforms = platforms.all;
+    license = licenses.asl20;
```
it probably make sense to reuse the maintainer of selenium here.

Mic92 commented on pkgs/development/tools/selenium/htmlunit-driver/default.nix:
```diff
@@ -0,0 +1,29 @@
+
+  unpackPhase = "true";
+
+  buildInputs = [ ];
```
this line can be removed.

Mic92 commented on pkgs/development/tools/selenium/htmlunit-driver/default.nix:
```diff
@@ -0,0 +1,29 @@
+
+  installPhase = ''
+    mkdir -p $out/share/lib/${name}
+    cp $src $out/share/lib/${name}/${name}.jar
```
```
installPhase = "install -D $src $out/share/lib/${name}/${name}.jar";
```

hsenag committed:
selenium-server-standalone: restore htmlunit-driver support
It was moved to a separate project between versions 2.45 and 2.53:
https://github.com/SeleniumHQ/selenium/commit/2d3150b

hsenag commented:
@Mic92 thanks, I've updated the commit with your suggestions.
